torchci/pages/api/drci/drci.ts

```diff
@@ -41,25 +53,113 @@ export async function updateDrciComments(octokit: Octokit, prNumber?: string) {
     );
 
     const workflowsByPR = reorganizeWorkflows(recentWorkflows);
+    await addMergeBaseCommits(octokit, workflowsByPR);
     const sevs = getActiveSEVs(await fetchIssuesByLabel("ci: sev"));
+    const masterFlakyJobs: FlakyJob[] = await fetchJSON(FLAKY_RULES_JSON) || [];
```

Comment:
Could we consider using a different name for `FlakyJob` dict and `masterFlakyJobs` array here?  The main reason is that they are just the list of flaky rules, which is not the same as the GitHub job `RecentWorkflowsData` used in `PRandJobs`.  So it's easy to confuse between the twos

Reply:
renamed a bunch of stuff to flakyrules